pkgs/development/tools/casperjs/default.nix

```diff
@@ -28,9 +28,9 @@ in stdenv.mkDerivation rec {
     make test
   '';
 
-
   installPhase = ''
-    mv $PWD $out
+    mkdir $out
```

Comment:
I'm aware that `rename()` is faster, but the initial issue description states that the PR is intended to solve problems that occur when `/nix` and `/tmp` are on *different* mount points, so this seems irrelevant. Even in the best case scenario, it seems to me the benefit of `rename()` won't really matter unless you're pushing large amounts of data. All in all, I don't find the performance argument all that compelling.